en/02_Developer_Guides/03_Forms/How_Tos/05_Simple_Contact_Form.md

```diff
@@ -144,12 +144,11 @@ class ContactPageController extends PageController
 }
 ```
 
-[hint]
- Caution: This form is prone to abuse by spammers,
- since it doesn't enforce a rate limitation, or checks for bots.
- We recommend to use a validation service like the ["recaptcha" module](https://www.silverstripe.org/spam-protection-module)
- for better security.
-[/hint]
+> [!CAUTION]
+> Caution: This form is prone to abuse by spammers,
+> since it doesn't enforce a rate limitation, or checks for bots.
+> We recommend to use a validation service like the ["recaptcha" module](https://www.silverstripe.org/spam-protection-module)
+> for better security.
```

Comment:
Intentionally changed to caution because based on the content that's obviously more appropriate